### ISSUE TITLE
Include tango-examples-c in readme

### ISSUE THREAD

**Issue by mihaibujanca** [stetro/project-tango-poc#7]
Hello,

You're also referencing tango-examples-c in your Android.mk, it'd be great if you could link that under libraries as well. I already had it installed but it might help others.

Thanks

**Comment by stetro** [comment]
Hey thanks for the feedback! I will clean this up, when I've finished my thesis in two weeks :)

**Comment by mihaibujanca** [comment]
Great! Good luck with your thesis, I hope it goes well!

**Comment by MatthewGitter** [comment]
Hello. 
Building the project in Android Studio requires OpenCV sources (and others, as per README.md). Could you document how to populate and configure the development environment including those? Or else perhaps complete the Gradle configs to set them up along with the rest of the project. FWIW I'm building on Ubuntu 16.4 . Thanks.
- For example:
  .....................................................
  Error:(34) /home/stetro/Source/opencv/platforms/build_android_arm//OpenCV.mk: No such file or directory
  .....................................................
- Not included, as per README.md :
  .....................................................
  Build PCL Library
  Build open_chisel Library
  Build opencv+contrib Library
  .....................................................

BTW, congratulations on such an interesting and apparently successful thesis project.

**Comment by mihaibujanca** [comment]
@MatthewGitter You'll need to modify some of the paths (e.g  **/home/stetro/Source/opencv/platforms/build_android_arm//OpenCV.mk** is an absolute path @stetro used on his own machine)

**Comment by MatthewGitter** [comment]
Yes, I saw that. But the OpenCV sources aren't included in this repo, nor the others mentioned in README.md as "not included". So I'm looking for instructions for completing that build environment, including the sources and other dependency content, for everything missing from the repo. Or even better, completing them in the repo and configuring its Gradle files to use/maintain them.
